redesign/test.py

```diff
@@ -0,0 +1,6 @@
+import dataprofiler as dp
+import generator_builder as gb
+
+profile = dp.Profiler(dp.Data("./test.csv"))
+data = gb.Generator(profile).synthesize()
```

Comment:
then update this to be `synthetic_data`
However, by doing this in a single line, are we not losing the generator?
so should we do:
```python
synth_gen = gb.Generator(profile)
synth_data = synth_gen.synthesize()?
```